terraform/aws/security/aws-network-acl-allows-all-ports.yaml

```diff
@@ -29,9 +29,12 @@ rules:
               action = "allow"
               ...
             }
-  - metavariable-regex:
-      metavariable: $PROTOCOL
-      regex: \A("all"|"-1"|-1)\Z
+      - metavariable-regex:
+          metavariable: $NETWORK_ACL
+          regex: ^(aws_network_acl|aws_default_network_acl)$
+    - metavariable-regex:
+        metavariable: $PROTOCOL
+        regex: \A("all"|"-1"|-1)\Z
```

Comment:
Ah one more thing, you need to un-indent the last `metavariable-regex` as a whole (L35-37) by 2 spaces

Reply:
This is what I get for making changes in the GitHub editor! 😅 
Fixed, sorry about that.

Reply:
All good!!